### PR TITLE
Changes to enable Sysbox builds for Flatcar OS.

### PR DESCRIPTION
**Pull request by ctalledo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ SYSRUNC_DIR     := sysbox-runc
 SYSFS_DIR       := sysbox-fs
 SYSMGR_DIR      := sysbox-mgr
 SYSIPC_DIR      := sysbox-ipc
-SYSLIBS_DIR     := sysbox-libs
+sSYSLIBS_DIR     := sysbox-libs
```
Comment (rodnymolina) [inline Makefile:42]: typo

File: tests/scr/flatcar/build-shiftfs [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```
Comment (rodnymolina) [inline tests/scr/flatcar/build-shiftfs:1]: We should parametrize and move this script to a centralized location to allow users to build shiftfs module by themselves -- something for later.

**Review by rodnymolina — APPROVED** [review]
Very clean, specially taking into account how much of a "one-of-a-kind" flatcar is when compared to our other supported distros.

File: Makefile (hunk 1 above)
Comment (ctalledo) [reply Makefile]: Fixed, thanks.

File: tests/scr/flatcar/build-shiftfs (hunk 2 above)
Comment (ctalledo) [reply tests/scr/flatcar/build-shiftfs:1]: Agreed ... future change.

**Pull request merged**